src/app/use-cases/validate-document.ts

```diff
@@ -68,5 +71,10 @@ export const makeValidateDocument =
         )
```

Comment:
may you add a comment on what the method 'makeValidateDocument' is supposed to do? explaining why we're putting the message into the queue.